### PR TITLE
Add Statamic v4 and Laravel 10 support

### LINKED ISSUE #3: Statamic 4 / Laravel 10 support
Are there any news regarding Statamic 4 / Laravel 10 support?

### PR DESCRIPTION
**Pull request by CraigBowler**
- Update minimum PHP version to 8.2
- Add support for Statamic v4
- Add support for Laravel 10
- Update various things to cater for Sendinblue -> Brevo name change such as:
  - Config form UI labels
  - Config file name
  - Namespacing
- Removed deprecated Sendinblue PHP SDK dependency and replaced with Brevo PHP SDK

### CODE REVIEW

File: src/Fieldtypes/SibAttribute.php
Comment (MeMattStone) [inline src/Fieldtypes/SibAttribute.php:1]: Do we change `SibAttribute.php` and `$sib_attribute` to be `BrevoAttribute.php` and `$brevo_attribute`, or just leave as they are?

File: src/Fieldtypes/SibList.php
Comment (MeMattStone) [inline src/Fieldtypes/SibList.php:1]: Do we change `SibList.php` and `$sib_list` to be `BrevoList`.php and `$brevo_list`, or just leave as they are?

File: src/Fieldtypes/SibAttribute.php
Comment (CraigBowler) [reply src/Fieldtypes/SibAttribute.php:1]: Yes I think we probably should, I'll update the PR.

File: src/Fieldtypes/SibList.php
Comment (CraigBowler) [reply src/Fieldtypes/SibList.php:1]: Yeah we should, I'll update the PR.

File: src/Fieldtypes/SibAttribute.php
Comment (CraigBowler) [reply src/Fieldtypes/SibAttribute.php:1]: @MeMattStone I have pushed changes for this now 😄

File: src/Fieldtypes/SibList.php
Comment (CraigBowler) [reply src/Fieldtypes/SibList.php:1]: @MeMattStone I have updated this now

**Comment by MeMattStone** [comment]
When testing on a fresh installation I'm getting the following error

Screenshot 2024-02-13 at 00 09 49

It's referencing sendinblue config, but think it should be brevo due to the changes made to names

**Pull request closed without merging**